src/app/qgsdxfexportdialog.cpp

```diff
@@ -797,6 +809,283 @@ void QgsDxfExportDialog::deselectDataDefinedBlocks()
 }
 
 
+void QgsDxfExportDialog::loadSettingsFromFile()
+{
+  const QString fileName = QFileDialog::getOpenFileName( this, tr( "Load DXF Export settings" ),
+                           QgsDxfExportDialog::settingsDxfLastSettingsDir->value(),
+                           tr( "XML file" ) + " (*.xml)" );
+  if ( fileName.isNull() )
+  {
+    return;
+  }
+
+  bool resultFlag = false;
+
+  QDomDocument domDocument( QStringLiteral( "qgis" ) );
+
+  // location of problem associated with errorMsg
+  int line, column;
+  QString errorMessage;
+
+  QFile file( fileName );
+  if ( file.open( QFile::ReadOnly ) )
+  {
+    QgsDebugMsgLevel( QStringLiteral( "file found %1" ).arg( fileName ), 2 );
+    // read file
+    resultFlag = domDocument.setContent( &file, &errorMessage, &line, &column );
+    if ( !resultFlag )
+      errorMessage = tr( "%1 at line %2 column %3" ).arg( errorMessage ).arg( line ).arg( column );
+    file.close();
+  }
+
+  if ( QMessageBox::question( this,
+                              tr( "DXF Export load from XML file" ),
+                              tr( "Are you sure you want to load settings from XML? This will change some values in the DXF Export dialog." ) ) == QMessageBox::Yes )
+  {
+    resultFlag = loadSettingsFromXML( domDocument, errorMessage );
+    if ( !resultFlag )
+      QMessageBox::information( this, tr( "Load DXF settings" ), tr( "ERROR: Failed to load DXF Export settings file as %1.\n\n%2" ).arg( fileName, errorMessage ) );
```

Comment:
```suggestion
      QMessageBox::information( this, tr( "Load DXF Settings" ), tr( "ERROR: Failed to load DXF Export settings file as %1.\n\n%2" ).arg( fileName, errorMessage ) );
```
missing "export" deliberate?